src/common/module-type.ts

```diff
@@ -14,6 +14,7 @@ export enum ModuleType {
   MARKET = 6,
   DROP = 7,
   DATASTORE = 8,
+  ROYALTY = 9,
```

Comment:
I think this has to line up with something in the contracts? But I might be misremembering @jakeloo can u confirm?

Reply:
nope, the contract is now agnostic. any integer you pass in is going to be valid. :)

Reply:
sheesh ok